app/jobs/ping_organization_webhook_job.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class PingOrganizationWebhookJob < ApplicationJob
+  queue_as :github_webhook
+
+  def should_perform_job?(organization, user)
+    (organization.present? && user.present?) &&
+      (organization.webhook_id.present? && !organization.is_webhook_active?)
+  end
+
+  def perform(organization_id, user_id)
+    organization = Organization.find_by(id: organization_id)
+    user = User.find_by(id: user_id)
+
```

Comment:
I think we can condense this a little bit and take advantage of SideKiq's ActiveJob error handling https://github.com/mperham/sidekiq/wiki/Active-Job#customizing-error-handling

What do you think about this?

```ruby
# frozen_string_literal: true
class PingOrganizationWebhookJob < ApplicationJob
  queue_as :github_webhook

  rescue_from(GitHub::Error) do
    retry_job wait: 1.minute, queue: :github_webhook_failures
  end
  
  def perform(organization_id, user_id)
    organization = Organization.includes(:user).find(organization_id)
    user = organization.users.find(user_id)
  rescue ActiveRecord::RecordNotFound => error

    return if error.present?
    return if organization.webhook_id.present? && organization.is_webhook_active?

    organization.ping_organization_webhook(user.github_client)
  end
end